tests/MySqlConnector.Tests/MySqlConnectionStringBuilderTests.cs

```diff
@@ -58,6 +58,7 @@ public void Defaults()
 #if !BASELINE
 			Assert.Null(csb.ServerRsaPublicKeyFile);
 #endif
+			Assert.Null(csb.ServerSPN);
```

Comment:
These tests also run against Oracle's Connector/NET (to verify equivalent behaviour) so this line will need to be moved inside the `#if !BASELINE` block to avoid a compiler error with MySqlData.